infrastructure/utils/scripts/installation/prerequisites/install-k3s.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# This script is used to install k3s.
+
+curl -sfL https://get.k3s.io |sh -s - --write-kubeconfig-mode 644 --docker --write-kubeconfig ~/.kube/config
```

Comment:
```suggestion
curl -sfL https://get.k3s.io |sh -s - --write-kubeconfig-mode 644 --docker --write-kubeconfig ~/.kube/config --kubelet-arg cgroup-driver=systemd
```
This doesn't change anything for Ubuntu installs, but is mandatory for Manjaro for example